client/src/components/ProjectForm.js

```diff
@@ -0,0 +1,163 @@
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+
+    setFormData((fData) => ({
+      ...fData,
+      [name]: value,
+    }));
+    setFormErrors([]);
+  };
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    const projectApi = new ProjectApiService();
+
+    try {
+      console.log('FORM DATA', formData);
+      console.log('PROJECT API SERVICE', projectApi.create);
+      const response = await projectApi.create(formData);
+      console.log('RESPONSE', response);
+    } catch (errors) {
+      setFormErrors(errors);
+      return;
+    }
+
+    //For now, navigate to projects page after successful form submission
+    //TODO - Figure out how to navigate to newly created project details page
+    history.push('/projects');
+  };
+
+  /** All TextField and InputLabel styles are controlled by the theme */
+
+  const simpleInputs = [
+    {
+      label: 'Project Name',
+      name: 'name',
+      type: 'text',
+      placeholder: 'Enter project name',
+    },
+    {
+      label: 'Project Description',
+      name: 'description',
+      type: 'textarea',
+      placeholder: 'Enter project description',
+    },
+    {
+      label: 'Location',
+      name: 'location',
+      type: 'text',
+      placeholder: 'Enter project location',
+    },
+    {
+      label: 'GitHub Identifier',
+      name: 'githubIdentifier',
+      type: 'text',
+      placeholder: 'Enter GitHub identifier',
+    },
+    {
+      label: 'GitHub URL',
+      name: 'githubUrl',
+      type: 'text',
+      placeholder: 'htttps://github.com/',
+    },
+    {
+      label: 'Slack URL',
+      name: 'slackUrl',
+      type: 'text',
+      placeholder: 'htttps://slack.com/',
+    },
+    {
+      label: 'Google Drive URL',
+      name: 'googleDriveUrl',
+      type: 'text',
+      placeholder: 'htttps://drive.google.com/',
+    },
+    {
+      label: 'HFLA Website URL',
+      name: 'hflaWebsiteUrl',
+      type: 'text',
+      placeholder: 'htttps://hackforla.org/projects/',
+    },
+  ];
+
+  return (
+    <Box>
+      <Box sx={{ textAlign: 'center' }}>
+        <Typography variant="h1">Project Management</Typography>
+      </Box>
+      <Box sx={{ bgcolor: '#F5F5F5' }}>
+        <Box sx={{ p: 2 }}>
+          <Typography sx={{ fontSize: '18px', fontWeight: '600' }}>
+            Create New Project
+          </Typography>
+        </Box>
+        <Divider sx={{ border: '1px solid black' }} />
+        <Box sx={{ py: 2, px: 4 }}>
+          <form onSubmit={handleSubmit}>
+            {simpleInputs.map((input) => (
+              <Box sx={{ mb: 3 }} key={input.name}>
+                <InputLabel id={input.name}>{input.label}</InputLabel>
+                <TextField
+                  id={input.name}
+                  name={input.name}
+                  placeholder={input.placeholder}
+                  variant="outlined"
+                  type={input.type}
+                  onChange={handleChange}
```

Comment:
Suggest adding something along the lines
```
helperText=" "
```
So that when we get to form validation, there isn't any layout shift when errors occur

<img width="468" alt="image" src="https://user-images.githubusercontent.com/5898009/224125797-ef348577-dded-4bc2-ac06-4c560fe56246.png">
<img width="468" alt="image" src="https://user-images.githubusercontent.com/5898009/224125834-29ebef7d-1b41-4e22-8b81-046958695c93.png">

Reply:
@Spiteless 

My plan was to leave the form as simplistic and skeleton like as possible in order to allow whoever works on the form validations to make these decisions

I appreciate your foresight and suggestion for sure :+1: